### ISSUE TITLE
"Enable GPS" nag when GPS is off can't be dismissed

### ISSUE THREAD

**Issue by somehundred** [Zverik/every_door#800]
If location is turned off, EveryDoor shows "Enable GPS" nag every time app is opened and directs to Android settings when "OK" or back button tapped. There is no option to dismiss it or to disable the nag in settings entirely.

It is very annoying.

<details>
<summary>Screenshot</summary>

![Screenshot_20241005_114459_Every Door](https://github.com/user-attachments/assets/e10bcc9c-9b9e-466f-af85-982fba1d501e)

</details>

**Comment by mnalis** [comment]
> There is no option to dismiss it

For me, when I  press `OK` in that screenshot, it open GPS properties, but when I press **Android `back` icon**, it returns back to Every Door and I can use it normally - such warning never bothering me again (even if I leave the app and come back to it), until such time I manually remove EveryDoor from list of recent Android apps (or reboot the phone).

Does that work for you?

**Comment by mnalis** [comment]
> This isn't mistake. It actually opens Android settings even if I press the back button.

Yes, that seems like a (minor) unexpected annoyance. But when GPS settings do get open, you can click back, right?

> yes, it does not show up if I leave EveryDoor open in background but I tend to always close my recent apps

As a workaround for that, depending on your Android version, you might be able to prevent EveryDoor from being closed when you close all other apps, e.g. on my Android 14 choosing `Keep open` accomplishes  this:

![small_Screenshot_20241005_195844_One UI Home](https://github.com/user-attachments/assets/327ba274-c1ae-485a-b53b-d3a9af8f84de)
